Source/TaskDelegate.swift

```diff
@@ -242,9 +239,9 @@ class DataTaskDelegate: TaskDelegate, URLSessionDataDelegate {
             progress.completedUnitCount = totalBytesReceived
 
             dataProgress?(
-                bytesReceived: Int64(data.count),
-                totalBytesReceived: totalBytesReceived,
-                totalBytesExpectedToReceive: totalBytesExpected
+                Int64(data.count),
```

Comment:
Because of this new change, do you think it warrants creating a `bytesReceived` variable to make it a bit more clear?